examples/rules/domain.yml

```diff
@@ -20,8 +20,26 @@ slots:
   some_slot:
     type: text
     influence_conversation: false
+    mappings:
+      # Slot mappings can be defined in the domain.
+      # You can also implement custom slot mappings in your validate function for the
```

Comment:
Is this comment now correct if we're changing how custom slot mappings are implemented?

Reply:
This will probably have to be adapted after the rasa-sdk changes, because it refers to `FormValidationAction` and that interface will be changed to work outside Forms too. I will make a note about this comment in this file 👍🏼